src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -50,6 +50,10 @@ public function testCopyUnreadableFileFails()
             $this->markTestSkipped('This test cannot run on Windows.');
         }
 
+        if (\function_exists('posix_getuid') && 0 === posix_getuid()) {
```

Comment:
shouldn't you use `!getenv('USER') || 'root' === getenv('USER')` as a check instead?

Reply:
I thought about it but like I said in https://github.com/symfony/symfony/pull/28529#issuecomment-423517921 if we use this check, test will be always skipped if executed inside a container 

I don't know which one is the best in our case ?
`!getenv('USER') || 'root' === getenv('USER')` is used multiple time inside symfony so i guess it's better to keep things similar

Reply:
On the other side, the skip relies on `posix_getuid` which is not always available, so that people will still experiment with the failure if they don't have the extension.
I prefer a skip for some than a failure for some others, so let's go with the getenv variant.